src/pybind/ceph_argparse.py

```diff
@@ -362,7 +371,98 @@ def argdesc(self, attrs):
         if self.goodchars:
             attrs['goodchars'] = self.goodchars
         return super().argdesc(attrs)
+    
+
+class CephArgtypeConverter(CephArgtype):
+    def convert(self, value):  # noqa # pylint: disable=unused-variable
+        raise NotImplementedError()
+
+
+class CLI_ANNOTATIONS:
+    MemorySize = 'cli_memory_size'
+
+
+def _get_annotation(tp):
+    if get_origin(tp) is Annotated:
+        annotated_args = get_args(tp)
+        if len(annotated_args) >= 2:
+            metadata_type = annotated_args[1]
+            return metadata_type
+
+        
+def _get_annotated_type_converter(annotation) -> Optional[CephArgtypeConverter]:
+    if annotation == CLI_ANNOTATIONS.MemorySize:
+        return CephSizeBytes()
```

Comment:
I'd put these under the `CephArgType`, for the same reason (scope/encapsulation).

Reply:
problem is that I use `CephArgtypeConverter` in the type hints, which can't be used inside `CephArgType` since `CephArgtypeConverter` is derived from `CephArgType`. wdyt I should do?